src/AppBar/AppBar.js

```diff
@@ -1,19 +1,18 @@
 import React from 'react';
 import IconButton from '../IconButton';
 import NavigationMenu from '../svg-icons/navigation/menu';
-import getMuiTheme from '../styles/getMuiTheme';
 import Paper from '../Paper';
 import propTypes from '../utils/propTypes';
 import warning from 'warning';
 
-function getStyles(props, state) {
+function getStyles(context) {
```

Comment:
I agree. That was one of the first one I changed, and must have missed it in the tidy-up when I settled on a consistent signature.